shared/test/test_regexp.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'test_helper'
+
+require 'cdo/regexp'
+class RegexpUtilsTest < Minitest::Test
+  EXPECTED_DIGITS = '1234567890'
+  VALID_US_PHONE_NUMBERS = %w[
```

Comment:
Thanks @bcjordan. Here you go: https://github.com/code-dot-org/code-dot-org/pull/8746